### DISCUSSION TITLE
Using fzf and bash scripting to make fabric a bit easier to use

### DISCUSSION THREAD

**Discussion by Piotr1215** [danielmiessler/Fabric#1085] [Show and tell]
First, thank you for a great tool. It's a real time saver for many repeatable tasks. 

To make it a little easier to use, I've created [a simple script](https://github.com/Piotr1215/dotfiles/blob/master/scripts/__orchestrator.sh) to orchestrate fabric commands. 

Here is a video about it if anyone is interested.

[![fabric automation](https://img.youtube.com/vi/_oKv8uV0Sz4/maxresdefault.jpg)](https://youtu.be/_oKv8uV0Sz4)

**Comment by gchingon** [comment]
this is a little over my head, but I think I get it. I tinkered with LLMs for a few days to run fabric -y "$URL" -p "pattern" > "$video-title-$id-$pattern.md" and also download the video. but this would be great, if I understand it correctly, to get multiple pattern-markdown files (after saving) and / or distilling one pattern into another and another until I get a full result I wish.

**Reply by Piotr1215** [reply]
Yes, that's the idea behind the script. You feed results of a pattern to another (or the same again) and once you're happy, just output the final result in a temporary `neovim` buffer. And since we are using sessions in fabric, every subsequent pattern has context about previous ones.
